### PR TITLE
changed shocklink.net to openshock.app in all guides

### PR DESCRIPTION
**Pull request by Kyobinoyo**

### CODE REVIEW

**Comment by LostQuasar** [comment]
OpenShock should be capitalized as such

File: mkdocs.yml
Comment (hhvrc) [inline mkdocs.yml:1]: Can you rename the shockink-* files as well while ur at it? ^^

File: mkdocs.yml [hunk 1]
```diff
@@ -14,17 +14,12 @@ nav:
       - guides/openshock-how-to-flash-your-board.md
       - guides/openshock-first-setup.md
       - guides/openshock-how-to-update.md
-    - Shocklink:
```
Comment (LostQuasar) [inline mkdocs.yml:17]: Maybe this should be OpenShock App and the previous category can be OpenShock Hardware?

File: mkdocs.yml
Comment (Kyobinoyo) [reply mkdocs.yml:1]: uh sure

File: mkdocs.yml (hunk 1 above)
Comment (Kyobinoyo) [reply mkdocs.yml:17]: doesn't really makes sense to me

Comment (hhvrc) [reply mkdocs.yml:17]: This is not needed, as the guides in the openshock and shocklink groups both cover frontend functionality, think its ok to merge them

**Review by hhvrc — APPROVED** [review]

**Pull request closed without merging**